app/views/movies/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<div id="movie_show_<%= @movie.id %>">
+<h1><%= @movie.title %></h1>
+<h3>Movie year: <%= @movie.creation_year %></h3>
+<p>Genre: <%= "#{@movie.genre}" %></p>
+<p>Actors: <% @movie.actors.each do |actor| %> </br>
```

Comment:
The user story asked for these to be ordered by the actor's age, youngest to oldest, so we'd want to have an instance method on `Movie` to do that for us. Then, we could call that method here.